src/components/PlayButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { LargeButton } from '@/components/LargeButton'
```

Comment:
Please see the button behavior [here](https://vercel.live/open-feedback/cla-rwanda-git-bugfix-image-containers-jeremiahchienda.vercel.app?via=pr-comment-visit-preview-link&passThrough=1) for reference